script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_9.txt

```diff
@@ -50,7 +50,7 @@ Because it's clear that the person in front of me isn't a bad person at all. The
 In any case, if I were to believe what Arcueid says, then I have no need to worry about being killed at random.
 That's why I feel safe... I feel safe, but more so than that important fact, there's something else I'm relieved about.
 <Page14>
-...Geez, what am I thinking.
+...Geez, what am I thinking?
 To think that my feelings would be so misdirected that I'd be happy about the fact Arcueid is only a half-vampire.
```

Comment:
To think that my feelings are so messed up that I'd be happy to know that Arcueid is only a half-vampire.

This is where I'd put my italis, IF I HAD THEM
![image](https://user-images.githubusercontent.com/16092757/141877356-87275ce8-68b1-4950-9fa8-ff3b0b9dc1f2.png)